legend-engine-xt-relationalStore-executionPlan-connection/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/connection/ds/DataSourceSpecification.java

```diff
@@ -56,6 +56,7 @@
     // Information parameters and its defaults: https://github.com/brettwooldridge/HikariCP#gear-configuration-knobs-baby
     public static final String HIKARICP_HOUSEKEEPING_PERIOD_MS = "com.zaxxer.hikari.housekeeping.periodMs";
     protected static final int HIKARICP_DEFAULT_MAX_POOL_SIZE = 100;
+    protected static final int HIKARICP_LEAK_DETECTION_THRESHOLD = 60 * 1000;
```

Comment:
I think 60 seconds is too low, specially when the wait time for a new connection is 30 seconds.  I will say we should have it as 5 or 10 times the wait time.  Otherwise, we might see too many false positives.